lib/dotcom_web/components/trip_planner/input_form.ex

```diff
@@ -108,6 +104,32 @@ defmodule DotcomWeb.Components.TripPlanner.InputForm do
     """
   end
 
+  defp location_search_box(assigns) do
+    assigns =
+      assigns
+      |> assign(:location_keys, InputForm.Location.fields())
+
+    ~H"""
+    <fieldset class="mb-sm -mt-md" id={"#{@name}-wrapper"}>
+      <legend class="text-charcoal-40 m-0 py-sm">{Phoenix.Naming.humanize(@field.field)}</legend>
+      <.algolia_autocomplete config_type="trip-planner" placeholder="Enter a location" id={@name}>
+        <.inputs_for :let={location_f} field={@field} skip_hidden={true}>
+          <input
+            :for={subfield <- @location_keys}
+            type="hidden"
+            id={location_f[subfield].id}
+            value={location_f[subfield].value}
+            name={location_f[subfield].name}
+          />
+        </.inputs_for>
+        <.feedback :for={{msg, _} <- @field.errors} :if={used_input?(@field)} kind={:error}>
```

Comment:
This change is creating some spurious errors (as soon as you enter an origin, it complains that there's no destination, and vice versa), but it's also conflicting with `main`. I wonder if resolving the conflicts with `main` would fix this.

(I think resolving the conflicts with `main` would actually just main using exactly this version of this file, but with the new "Enter [an origin|a destination] location" placeholders, since the big change to `main` was re-adding the error feedback component, which you also added here.)

Reply:
I think resolving the conflict fixed it?!? Feel free to try again :)